roles/performance_profile_nrop/tasks/main.yml

```diff
@@ -0,0 +1,86 @@
+---
+- name: Get node pool
+  kubernetes.core.k8s_info:
+    api_version: hypershift.openshift.io/v1beta1
+    kubeconfig: "{{ pp_kubeconfig | default(omit) }}"
+    kind: NodePool
+    namespace: "{{ nodepool_namespace }}"
```

Comment:
Assuming you make the changes to defaults/main.yml as mentioned above then  the above needs changed to:
`namespace: "{{ performance_profile_nrop_np_namespace }}"`